src/main/java/com/dumbdogdiner/stickyapi/common/util/textures/Mobhead.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.common.util.textures;
+
+import com.dumbdogdiner.stickyapi.common.util.StringUtil;
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Credit: All mob heads are borrowed from the vanillatweaks datapacks.
+ * @see <a href="https://vanillatweaks.net/">Vanilla Tweaks</a>
+ */
+@SuppressWarnings("unused")
+public enum Mobhead {
+    BAT("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvOGEyOWM5ZWNlNDI4ZmEyMzM5NWFjMjAxOWJmMmQwMjc0NDA1MjlmMjUzM2ZjODIwMWU3YjNkYTBmNjBmMjAwNSJ9fX0="),
```

Comment:
Don't tell me these are base 64 encoded pngs...

Reply:
Its better (worse), its json of the texture URLs, i plan to make it read from a datapack but zip and stuff is a pain in the ass

Reply:
@SkyezerFox you innocent thing, if it were PNGs they'd be bigger 🤣

Reply:
iVBORw0KGgoAAAANSUhEUgAAACgAAAAXCAMAAABODP0nAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAB2lBMVEVYWFhcXFxycnJpaWltbW1sbGx0dHR9fX2Pj4+bm5uampqjo6N7e3tzc3NWVlYoKCgAAAACAgIMDAwFBQUJCQkXFxdhYWFdXV1lZWWGhoaVlZWZmZmXl5eOjo53d3clJSWFhYVGRkY/Pz9AQEBVVVVaWlpoaGhxcXFwcHA+Pj4VFRUBAQEHBwcGBgY6OjpPT09bW1uBgYFra2s3NzepqamQkJAyMjIzMzMxMTFBQUE5OTk7OzspKSmHh4ccHBwvLy9HR0cmJiYKCgoODg4iIiIbGxshISEYGBiEhISnpqd5eXk9PT1NTU1QUFBXV1dMTExTU1OgoKCqqqpqamoDAwNSUVJUVFQWFhZOTk4fHx8qKiqAgICcnJyNjY2WlpaoqKgICAisrKywsLDZ2dkQEBA2NjZCQkJiYmJjY2NkZGSCgoKJiYmhoaHU1NS6urrHx8dvb29ISEhfX19mZmZeXl6ioqKlpaWysrIwMDAkJCR/f3+UlJQtLS2IiIiTk5M0NDQjIyNERERLS0tFRUUdHR12dna9vb16enoTExMPDw9JSUmMjIwrKysaGho4ODgsLCx+fn7c3Nytra0eHh4RERGLi4sNDQ14eHjAwMC2tra/v7/Nzc3Pz8////+D+YBAAAAAAWJLR0SdBrvysQAAAAd0SU1FB+QMCRIPB/+nr80AAANkSURBVCjPNdD5U9poAMbxNyRRUAKUIwREBIEQVAiHQhGjeGBcCoFtxI14tV3EgN3dJGA9AJXaViq9tsfef+yqM/3+/Jl5Zh4AIBWMoPDAoFozpNYMD2kxnd7wwGA0PTBbENxKYDb7iGNk1AnGdC4V7h73uL1WHwkTKj8VMJonDIbJqaAhRNBhnNJqXRF/FMQGpmeIeOJhYjw5m5qLMzZ03rSQji8uBaeWV8hkZnVshWUzbAhAaz+MZh/l8nnAFYqQ78fH/LqlpMXIjZ8MQYNQWl9iVzfL+s0i2HJs72Tpoi5MIiSl2trd8wWfjDx99rO2XAmaQrrUftV7YBVr9Xs4WNT5an6nP6s9fO6wTpp3f/n1N7V0YDKw46xH9loVouG6h3UANb/DI2KyvPdC/ewYU0nRk8wpXfJyPgU/u4c2SBG/T7eG2m0plRYEHO+4oIkU4LxFYcOLAazzYpdpksRdkgo5t82Q0MXY6hirC2fT85uXbihfkAUhCnRiZHBN5Aj4tijFH9p2Bmrd2Q1yZIeK3MFcnnt5JRQOwMVS5VX3NYxKt8EIsdVhGMn+xvF4eku8voNhnRW4XxYUYDH21j1agKpuQ3lxkFlrOf07PKngKHIHC3S1VCjICpgwmt/mXot8uNbAKVSp22x1xC/iPFHDldBbC5vZXHCnb5I6UDEbJ0BYYWYjDuoqn0I6fez2NlC1FruzSsASe/e+fXPSPikB89RyJtzoSlS/ZUt1r1Rqu7XgydFXsQ+hFIy3xz8aWbmtbxeByVTZFNE+MtpvHT1dc9VbGiEgpOKxUGJupXfjR2PzetCEuChASCv1SUv4cHiQ+f0z01F3pNN3+leZ0y+9r+akwImJR7Uoz0dBRCU6pQHEp6tubNjPpINrIqsvP5zbb1fWe72LdkhuyHk6T9OA997A0UbTiqQCSeGi+k2gicW4fzzgCf5R6XUv2bESCqppOQ14wK0d9rVwk95ftXy8bMT/BJ4yoKtV/ZP3H7xNi7HcxECRA0Dx5lyO46N+3R4Pn0WxEPjrHBmlXB247vy23WKccV/ExVBEFMCNA7vms6Yz01jmuRwHzq9xhp/+e3g7WyfxphtjkzwiE9lrgGLY0fO9mfrwJ/OXr4tB+Tjskz3SP//+x1cTCyuLpxd5ApLDA53/AYjB4uCzaqzoAAAAJXRFWHRkYXRlOmNyZWF0ZQAyMDIwLTEyLTA5VDE4OjE1OjA1KzAwOjAwcqdcowAAACV0RVh0ZGF0ZTptb2RpZnkAMjAyMC0xMi0wOVQxODoxNTowNSswMDowMAP65B8AAAAASUVORK5CYII=

okay maybe you're right